go.mod

```diff
@@ -4,43 +4,49 @@ go 1.18
 
 replace github.com/spf13/cobra => github.com/spf13/cobra v1.4.0
 
+replace "github.com/ONSdigital/dp-renderer" => "/users/work/documents/dp-renderer"
```

Comment:
The build is failing because this line needs to be removed

Reply:
I can see this has been resolved, resolving conversation